aries_vcx/tests/test_pool.rs

```diff
@@ -103,6 +106,37 @@ mod integration_tests {
         .await;
     }
 
+    #[tokio::test]
+    #[ignore]
+    async fn test_pool_write_new_endorser_did() {
+        SetupPoolDirectory::run(|setup| async move {
+            let faber = create_faber_trustee(setup.genesis_file_path.clone()).await;
+            let mut acme = create_faber(setup.genesis_file_path.clone()).await;
+            let acme_vk = acme.get_verkey_from_wallet(acme.public_did()).await;
+            info!(
+                "test_pool_write_new_endorser_did >> Faber trustee did: {}, acme did: {}, acme vkey: {}",
+                faber.public_did(),
+                acme.public_did(),
+                acme_vk
+            );
```

Comment:
The test would be easier to read if not "cluttered" with logs used for debugging.